domains/Accounts/Notifications/VerifyEmailNotification.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Domains\Accounts\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\Facades\URL;
+
+class VerifyEmailNotification extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param mixed $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param mixed $notifiable
+     * @return MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        $hash = \base64_encode(Hash::make($notifiable->getEmailForVerification()));
+        $link = URL::route(
+            'accounts.verify',
+            [
+                'id' => $notifiable->getKey(),
+                'hash' => $hash,
+            ]
+        );
+
+        return (new MailMessage)
+            ->subject(Lang::get('Verify Email Address'))
```

Comment:
Nit: we can use the `__()` helper method here.

Reply:
done